tests/test_openapi.py

```diff
@@ -45,3 +56,452 @@ def test_str2bool():
 
     default['ogc_schemas_location'] = '/opt/schemas.opengis.net'
     osl = get_ogc_schemas_location(default)
+
+
+def get_test_file_path(filename):
+    """
+    Helper function to open test file safely
+
+    :param filename: file path
+
+    :returns: corrected file path
+    """
+
+    if os.path.isfile(filename):
+        return filename
+    return 'tests/{}'.format(filename)
+
+
+@pytest.fixture()
+def config():
+    """
+    Get the OpenAPI Document configuration
+
+    :returns: OpenAPI configuration YAML dict
+    """
+    with open(get_test_file_path('pygeoapi-test-openapi-config.yml')) as config_file: # noqa
+        return yaml_load(config_file)
+
+
+@pytest.fixture()
+def get_oas_30_(config):
+    """
+    Get the generated OpenAPI 3.0 Document
+
+    :param config: configuration object
+
+    :returns: OpenAPI definition YAML dict
+    """
+    return get_oas_30(config)
+
+
+@pytest.fixture()
+def is_cql(config):
```

Comment:
@FarheenB  liked very much that you used `is_cql` as fixture

Reply:
thank you :)